### ISSUE TITLE
File Loss

### ISSUE THREAD

**Issue by DrugD** [shihhsinwang0214/SCHull#1]
You may loss a file:
xxx/SCHull/qm9_dataset.py

```
import os
import os.path as osp
import numpy as np
from tqdm import tqdm
import torch
from sklearn.utils import shuffle
import torch.nn.functional as F
from rdkit import Chem
from mol_unit_sphere import Frame
from torch_geometric.data import Data, DataLoader, InMemoryDataset, download_url, extract_zip

```

For:
`from mol_unit_sphere import Frame`
The mol_unit_sphere is loss.
